### PR TITLE
Make status descriptions and status context configurable

### PR DESCRIPTION
**Pull request by iradul**
Adds configuration page for the plugin where user can configure:
 * `context` of a status message
 * `description` of each status message

### CODE REVIEW

**Comment by iradul** [comment]
I just realized that there is already a pending PR for setting up the status context (#7), haven't checked it out though

**Comment by knownasilya** [comment]
I'm sure you can make the best of both worlds 👍

**Comment by iradul** [comment]
I swapped default context from **default** to **ci/strider** as you suggested in another PR, also did some bug fix for initial config page load.

**Comment by knownasilya** [comment]
I'll merge it once you say it's ready.

**Comment by iradul** [comment]
It's ready.

File: config/config.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        Configure context for status messages
+    </h3>
+    <div>context :</div>
+    <input type="text" class="span12" ui-refresh="selectedTab" ng-model="config.context" />
+    <br/>
+    <br/>
+    <h3 class="help-text">
+        Configure description for each status message
+    </h3>
+    <div>pending :</div>
```
Comment (knownasilya) [inline config/config.html]: These divs could probably be `<label>`s

Comment (knownasilya) [inline config/config.html]: See the config here https://github.com/Strider-CD/strider-node/blob/master/config/config.html

Comment (knownasilya) [inline config/config.html]: A minor nit-pick, `pending :` => `Pending: `

**Comment by knownasilya** [comment]
Excellent!

**Comment by knownasilya** [comment]
Released as 2.0 in the strider plugin interface.

**Pull request merged**